man/tpm2_nvrelease.1.md

```diff
@@ -0,0 +1,64 @@
+tpm2_nvrelease 1 "SEPTEMBER 2017" tpm2-tools
+==================================================
+
+NAME
+----
+
+tpm2_nvrelease(1) - Release a Non-Volatile (NV) index.
+
+SYNOPSIS
+--------
+
+`tpm2_nvrelease` [OPTIONS]
+
+DESCRIPTION
+-----------
+
+tpm2_nvrelease(1) - Release a Non-Volatile (NV) index that was previously
+defined with tpm2_nvdefine(1).
+
+OPTIONS
+-------
+
+  * `-x`, `--index`=_NV\_INDEX_:
+    Specifies the index to release.
+
+  * `-a`, `--authHandle`=_SECRET\_DATA\_FILE_:
+    specifies the handle used to authorize:
+    * `0x40000001` for `TPM_RH_OWNER`
+    * `0x4000000C` for `TPM_RH_PLATFORM`
```

Comment:
I think this is one of the last options that still require hex values as arguments, I've filled issue #512.